doc/plutus/tutorials/BasicPolicies.hs

```diff
@@ -30,7 +30,14 @@ oneAtATimePolicy _ ctx =
         minted = txInfoMint txinfo
     -- Here we're looking at some specific token name, which we
     -- will assume we've got from elsewhere for now.
-    in valueOf minted ownSymbol tname == 1
+    in case Map.lookup ownSymbol (getValue minted) of
+        Nothing -> n == 0
+        Just tokens -> all
+            (\(tname', n') -> if tname' == tname
+                then n' == 1
+                else n' == 0
+            )
+            (Map.toList tokens)
```

Comment:
If we merge https://github.com/input-output-hk/plutus/pull/4255 then we can replace this whole case 
```suggestion
in currencyValueOf ownSymbol minted == singleton ownSymbol tname 1
```